src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  background-color: rgb(214, 214, 214);
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  background-color: #00acdc;
+  height: 4px;
+  width: 37px;
+  border-radius: 8px;
+  position: relative;
+  top: auto;
+}
+
+.header {
+  min-width: 1024px;
+  max-width: 1200px;
+  display: flex;
+  align-items: center;
+  background-color: white;
+  justify-content: space-between;
+}
+
+.nav {
+  /* display: inline-block; */
+  height: 60px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin-right: 50px;
+  /* margin: 0; */
+  height: 60px;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  height: 20px;
+  line-height: 20px;
+  color: black;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
```

Comment:
Add the font family for the body tag to get it inherited by elements on the page

Reply:
Ready.